File: src/main/java/com/example/SpringDIExample.java
package com.example;

import org.springframework.context.support.GenericXmlApplicationContext;

import com.example.model.Dept;

public class SpringDIExample {

	public static void main(String[] args) {
		GenericXmlApplicationContext ctx = 
				new GenericXmlApplicationContext("classpath:/spring/beans.xml");
		
		Dept dept = ctx.getBean(Dept.class);
		System.out.println(dept);
		
		ctx.close();

	}

}

File: src/main/scala/com/example/scala/SpringDIExample.scala
package com.example.scala

import org.springframework.context.support.GenericXmlApplicationContext
import com.example.model.Dept

object SpringDIExample {
	def main(args: Array[String]): Unit = {
		var ctx = new GenericXmlApplicationContext("classpath:/spring/beans.xml")
		
		var dept = ctx.getBean(new Dept().getClass)
		println(dept)
		
		ctx.close()
	}

}